Automattic/jetpack
# VideoPress: Add support to poster image upload on video quick actions

Partial of #26500

#### Changes proposed in this Pull Request:

* Adds the `useLibraryPosterImage` hook allowing to select an image from the Media Library as the video's poster image from the video quick actions

Notes:
* Not implemented in the video details edit page yet.
* https://github.com/Automattic/jetpack/issues/26626 is still present

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to the VideoPress dashboard
* On the quick actions of a video, click on Update Thumbnail > Upload Image
* Select an image
* Wait for the image to be uploaded (TODO: propagate loading state to video card/row)
* Check that a request was made with the attachment id as `poster_attachment_id`

![2022-10-11_14-39-12](https://user-images.githubusercontent.com/8486249/195163425-d4aaaa5b-1d59-4e65-9979-6599a4220652.png)
